app/views/templates/_form.html.erb

```diff
@@ -84,6 +85,32 @@
       <% end %>
     </div>
 
+    <div class="tab-pane" id="safemode">
+      <table class="table table-striped table-fixed">
+        <thead>
+          <tr>
+            <th><%= _("Class") %></th>
+            <th><%= _("Allowed methods or members") %></th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td class="ellipsis"><em><%= _("Global methods (functions)") %></em></td>
+            <td><%= Foreman::Renderer::ALLOWED_HELPERS.join(' ') %></td>
+          </tr>
+          <tr>
+            <td class="ellipsis"><em><%= _("Global variables") %></em></td>
+            <td><%= Foreman::Renderer::ALLOWED_VARIABLES.map{|x| "@#{x}"}.join(' ') %></td>
```

Comment:
The list of variables (and helpers, above) should be sorted to match the jail methods.